### DISCUSSION TITLE
Portuguese translation

### DISCUSSION THREAD

**Discussion by PatriaMinas** [pawelsalawa/sqlitestudio#4565] [General]
Version 3.4.0 lacks some parts of the software available for translation into Portuguese.
I will translate when it's available on Crowdin. I also intend to improve some parts that didn't turn out well.

**Comment by pawelsalawa** [comment]
Please do. I intend to release 3.4.1 in a week or so.

**Comment by PatriaMinas** [comment]
Please look at the image below. I am not locating in Crowdin some parts of the software text, which are still in English.
I can not translate. I don't know how to do this in Crowdin.

What was available on Crowdin I translated today.

Thanks.

![translt](https://user-images.githubusercontent.com/95147314/203863857-95179aad-9baa-4c9d-96bd-a419563a002f.png)

**Reply by pawelsalawa** [reply]
Indeed, there's something wrong with the sync between CrowdIn and GitHub. Even manual synchronization triggering does not help. I need to look deeper into this.

**Reply by pawelsalawa** [reply]
Created #4580 to address this.

**Comment by PatriaMinas** [comment]
Very good! Everything that was available on Crowdin was translated into Brazilian Portuguese.
I know there are still some small details for me to improve the translation. I will be reviewing until the last moments before the next update 3.4.1 version.
Thanks for the solution @pawelsalawa.
